R/pkg/R/sparkR.R

```diff
@@ -537,9 +538,13 @@ clearJobGroup <- function() {
 #' cancelJobGroup("myJobGroup")
 #'}
 #' @note cancelJobGroup since 1.5.0
-cancelJobGroup <- function(groupId) {
+cancelJobGroup <- function(groupId, reason = NULL) {
```

Comment:
I'd suggest we don't add them as it's kind of low-level API and R/Python users shouldn't care.